DuckDuckGo/SettingsSubscriptionView.swift

```diff
@@ -102,12 +106,17 @@ struct SettingsSubscriptionView: View {
                                  subtitle: UserText.settingsPProITRSubTitle,
                                  action: { isShowingITP.toggle() }, isButton: true)
                 .sheet(isPresented: $isShowingITP) {
-                    SubscriptionITPView()
+                    SubscriptionITPView().onAppear(perform: {
+                        Pixel.fire(pixel: .privacyProIdentityRestorationSettings)
+                    })
                 }
             }
              
             if viewModel.shouldShowDBP || viewModel.shouldShowITP || viewModel.shouldShowNetP {
-                NavigationLink(destination: SubscriptionSettingsView()) {
+                NavigationLink(destination: SubscriptionSettingsView().onAppear(perform: {
+                    Pixel.fire(pixel: .privacyProSubscriptionSettings)
```

Comment:
Same as above.

Reply:
Subscription settings also has a single point of entry, so it would also make sense to put in the viewModel initializer for the view, or add an onAppear method there that's called from the view, like the ITP view does.   WDYT?